cicecore/cicedynB/dynamics/ice_transport_remap.F90

```diff
@@ -858,14 +844,7 @@ subroutine horizontal_remap (dt,             ntrace,   &
                                tm    (:,:,:,n,iblk))
 
             if (l_stop) then
-               write (nu_diag,*) 'istep1, my_task, iblk, cat =',     &
-                                  istep1, my_task, iblk, n
```

Comment:
and again here we loose the category info

Reply:
I am adding back the category output.